### ISSUE TITLE
A11y_api-catalog_Search_Keyboard: Keyboard only user is not able to see the whole content (Suggestion list items) since scroll bar is not working 

### ISSUE THREAD

**Issue by Amulya-95** [terrajobst/apisof.net#17]
[Check out Accessibility Insights](https://www.1eswiki.com/wiki/Accessibility_Insights) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS2.1.1; #Keyboard; #C+E-H; #api-catalog; #Edge; #Chrome; #A11ySev1; #Benchmark;

**Environment Details:**
URL: https://apisof.net/

**Browser:**
Microsoft Edge 44.18362.267.0
Chrome Version 76.0.3809.100

**Repro steps:**
1)Hit the URL : "http://apisof.net" and press enter
2)Tab till "Search" and press enter or else press "Slash" key
3)Tab till search field and type the content that you want to search
4)Navigate to suggestion list items using arrow key 
5)Check whether scroll bar is working by keyboard or not  

**Actual Result:**
When user navigate to suggestion list items using arrow keys, focus is going to content but keyboard only user is not able to see the content as scroll bar is not working

**Expected Result:**
When user navigate to suggestion list items using arrow keys, Scroll bar should work properly and keyboard only users should be able to see the content when focus lands on it

**User Impact:**
Keyboard only users will feel difficulty if the user is not able to see the content when focus lands on it

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Not able to see list items using keyboard as scroll bar is not working.pptx](https://github.com/terrajobst/apisof.net/files/3533655/Not.able.to.see.list.items.using.keyboard.as.scroll.bar.is.not.working.pptx)

**Comment by terrajobst** [comment]
The web site no longer exists.
